Components/Queues/Drivers/QueueDriverInterface.php

```diff
@@ -19,12 +20,12 @@ interface QueueDriverInterface extends SerializableInterface
     /**
      * Configures the driver.
      *
-     * @param $host
-     * @param string $username Username to connect to the queuing system
-     * @param string $password Password to connect to the queuing system
-     * @param string $format
+     * @param string $host
+     * @param string $username
+     * @param string $password
+     * @param string|null $vhost
      */
-    public function configure($host, $username, $password, $format = self::FORMAT_JSON);
+    public function configure(string $host, string $username, string $password, string $vhost = null);
```

Comment:
Which is the reason to add "vhost" in the queueDriver generic interface? For instance, ActiveMQ didn't have this concept

Reply:
It's a concept for the majority of the brokers and their connections. I think in the case of using the ActiveMQ, the default vhost "/" is acceptable, right?

Reply:
We can leave it as vhost null